metadata.yaml

```diff
@@ -8,4 +8,10 @@ tags:
 requires:
   reverseproxy:
     interface: http
-series: ['xenial']
\ No newline at end of file
+resources:
+    dhparam:
+        type: file
+        filename: dhparam.pem
```

Comment:
Me neither, further research in DHParams revealed this :stuck_out_tongue: The DHParam sets seem to be to low in size, 4096 is recommended at this time. And I think the cyclic ones are used in ECDH (elliptic curve DH) ciphers, but some used ciphers do not use these (correct me if i'm wrong), but this should change in the future I recon